trade-finance-manager-api/src/v1/__mocks__/mock-deal-MIA-second-submit-facilities-unissued-to-issued.js

```diff
@@ -194,6 +194,7 @@ const MOCK_DEAL_AIN_SUBMITTED_FACILITIES_UNISSUED_TO_ISSUED = {
         'coverEndDate-year': '2020',
         status: 'Submitted',
         ukefFacilityID: '65431',
+        requestedCoverStartDate: 1610369832226.0,
```

Comment:
We shouldn't hard code the timestamp. This timestamp is for 4/08/2021. Once this date passes all the tests that require requestedCoverStartDate to not be in the past will fail

Maybe change to `Date.now()`

Reply:
I should have learnt from before 😂 i'll update